sample_factory/enjoy.py

```diff
@@ -266,4 +266,6 @@ def max_frames_reached(frames):
         )
         push_to_hf(experiment_dir(cfg=cfg), cfg.hf_repository, cfg.num_policies)
 
-    return ExperimentStatus.SUCCESS, float(np.mean([np.mean(episode_rewards[i]) for i in range(env.num_agents)]))
+    return ExperimentStatus.SUCCESS, sum([sum(episode_rewards[i]) for i in range(env.num_agents)]) / sum(
```

Comment:
Would be more readable to have this rewritten as 2-3 lines of code with explicit variable names. But okay, as long as it works :)